Userland/Libraries/LibWeb/HTML/HTMLHyperlinkElementUtils.cpp

```diff
@@ -558,13 +559,17 @@ DeprecatedString HTMLHyperlinkElementUtils::get_an_elements_target() const
 bool HTMLHyperlinkElementUtils::get_an_elements_noopener(StringView target) const
 {
     // To get an element's noopener, given an a, area, or form element element and a string target:
+    auto rel = hyperlink_element_utils_rel();
+    auto link_types = rel.view().split_view_if(Infra::is_ascii_whitespace);
 
-    // FIXME: 1. If element's link types include the noopener or noreferrer
+    // 1. If element's link types include the noopener or noreferrer
     // keyword, then return true.
```

Comment:
Please indent wrapped comments to the level of their marker, i.e.

```
// 1. If element's link types include the noopener or noreferrer
//    keyword, then return true.
```

Arguably this one is short enough to not be wrapped at all.

Reply:
same below fwiw